packages/client/src/components/CheckSession.re

```diff
@@ -0,0 +1,46 @@
+let component = ReasonReact.statelessComponent("CheckSession");
+
+let make = _children => {
+  ...component,
+  render: _self => {
+    <Providers.AppContext.Consumer>
+      ...{({loggedInUser}) =>
+        switch (loggedInUser) {
+        | Some(loggedInUser) =>
+          <UserAccessTokenUpdate.Mutation>
```

Comment:
Is this running this mutation for all users, even if their email addresses and everything are already valid?

Reply:
Yes. And for what I am going to give an explanation below.